### PR TITLE
Added hash to invalidate cache for webpack bundle

### PR DESCRIPTION
**Pull request by kurtisassad**
## Description
I'm pretty sure Zak's issue stemmed from the fact that his browser was caching previous chunks from webpack. Added hash to webpack chunks on development (already enabled on production) to invalidate these when they are updated.

## Does this PR affect any server routes?
- [ ] yes
- [ ] no

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ ] yes

### CODE REVIEW

**Comment by CowMuon** [comment]
Good hypothesis Kurtis, makes sense since we didn't have that for Dev but only Prod.

**Review by CowMuon — APPROVED** [review]
LZGTM

File: packages/commonwealth/webpack/webpack.dev.config.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ module.exports = merge(common, {
   output: {
     publicPath: '/build',
     path: path.join(__dirname, '../build'),
-    filename: 'js/[name].js',
-    chunkFilename: 'js/[name].chunk.js',
+    filename: 'js/[name].[contenthash:8].js',
+    chunkFilename: 'js/[name].[contenthash:8].chunk.js',
```
Comment (zakhap) [inline packages/commonwealth/webpack/webpack.dev.config.js:26]: why is this not chunkhash like in `webpack.prod.config.js`?

Comment (zakhap) [reply packages/commonwealth/webpack/webpack.dev.config.js:26]: does it even matter?

**Pull request merged**